bionic/dagviz.py

```diff
@@ -154,7 +154,11 @@ def doc_from_node(node):
                 shape="box",
             )
             if doc:
-                tooltip = rewrap_docstring(doc)
+                tooltip = (rewrap_docstring(doc) + 
+                           f"\n\n Persisted: {graph.nodes[node].get('should_persist', '')}")
```

Comment:
Is it possible for the `should_persist` field to be missing? If so, when? If not, why do we need to call `get` with a default value?